Terminal.Gui/Terminal.Gui.csproj

```diff
@@ -39,8 +39,8 @@
   <!-- =================================================================== -->
   <ItemGroup>
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.3" PrivateAssets="all" />
```

Comment:
We can and probably should drop this reference, since V2 isn't .NET Framework-compatible.

The reference isn't used, anyway.